packages/bsky/src/db/migrations/20230927T202833377Z-create-moderation-subject-status.ts

```diff
@@ -0,0 +1,55 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_action')
+    .renameColumn('reason', 'comment')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .alterColumn('comment')
+    .dropNotNull()
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('refEventId', 'integer')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('meta', 'jsonb')
+    .execute()
+  await db.schema
+    .createTable('moderation_subject_status')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+    .addColumn('subjectType', 'varchar', (col) => col.notNull())
+    .addColumn('subjectDid', 'varchar', (col) => col.notNull())
+    .addColumn('subjectUri', 'varchar')
```

Comment:
it feels to me like these columns are a bit verbose in terms of database disk (and, in particular, index and pagecache) use.

eg, i'd probably break out URI in to separate collection and rkey columns (or just a combined `path` column), so that the DID doesn't need to be stored again. subjectType could be skipped/inferred based on whether `collection` is null or not (if null, it's a report on a repo, not a record). or an even bigger refactor would be separate tables for status-on-repo and status-on-record.

this is all kind of fiddly optimization and could wait till later. and defer to devin on more local style/norms in this repo.